SU2_CFD/include/fluid/CNEMOGas.hpp

```diff
@@ -42,7 +42,7 @@ class CNEMOGas : public CFluidModel {
   bool frozen,                           /*!< \brief Indicates if mixture is frozen. */
   ionization;                            /*!< \brief Presence of charged species in gas mixture. */
 
-  string Kind_GasModel;                  /*!< \brief String gas model. */
+  string gas_model;                      /*!< \brief String gas model. */
```

Comment:
the reason why I did this, it's because gas_model is actually a string, whereas in the SU2 convention the types Kind_Something from configure are usually integers associated to different models. So this is not the usual thing in SU2, that's why I did it differently to be consistent, it's really the name of the model

Reply:
let me know if this is ok with you

Reply:
Yup, that is fine.  I was misunderstanding!